src/Admin/SettingsPage/Campaign_Log_List.php

```diff
@@ -352,7 +352,12 @@ public function prepare_items()
     public function process_bulk_action()
     {
         // bail if user is not an admin or without admin privileges.
-        if ( ! current_user_can('administrator')) {
+        $capability = 'administrator';
+        if( current_user_can('manage_mailoptin')){
+            $capability = 'manage_mailoptin';
+        }
+
+        if ( ! current_user_can($capability)) {
```

Comment:
@picocodes please change every `administrator` capability to `manage_option`. Even though it works, it's not recommended. 

If we have a global function, we will eleminiate these if statements and thus simplify the codebase.